src/components/Navbar/NavbarMobile.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { FiMenu, FiLogOut } from 'react-icons/fi';
+import {
+  Avatar,
+  Button,
+  Flex,
+  HStack,
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'HStack' is defined but never used.